src/assets/content/dictionary/automation.md

```diff
@@ -1,6 +1,10 @@
 ---
 title: automation
-definition: ''
-perspectives: []
+definition: Automation is a term for technology applications where human input is minimized. This includes business process automation (BPA), IT automation, personal applications such as home automation and more.
+sources:
+- sourceurl: https://www.ibm.com/topics/automation
+perspectives:
+- meaning: Thing Which Minimize Human Effort.
```

Comment:
```suggestion
- meaning: Things Which Minimize Human Effort
```
Suggestion: I've suggested to remove the period, since one is already added in our code. I've also suggested to pluralize the word "Things" for grammar.

Reply:
Thanks for review , 
I will do that definately 😃